lev-data/drop_not_nulls.sql

```diff
@@ -0,0 +1,8 @@
+/* These columns had not null constraints, but the corresponding columns in the
+   data json had null values. Removed constraints to keep new data consistent.
+ */
```

Comment:
Thought this might be worth a comment so we can come back to it after chatting to hmpo